rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/TestSchemaNameCustomizable.java

```diff
@@ -66,7 +65,7 @@ void checkCustomSchemaNameIsDefined_Postgres() throws SQLException {
   @Test
   void checkCustomSchemaNameIsDefined_OtherDb() throws SQLException {
     resetDb();
-    Assume.assumeFalse("Test only executed if NOT Postgres", isPostgres);
```

Comment:
This is not an assertions. This is an assumption.
Please use `Assumptions.assumeThat(isPostgres).isFalse();` (with a static import)